Documentation/admin/upgrade.md

```diff
@@ -15,6 +15,23 @@ Please note that the update payload process may affect any or all components in
 
 To learn more about how this process is executed, read about [Operators][operators].
 
+## Production and pre-production channels
+
+Tectonic clusters can be conifigured to track either a production or pre-production update channel for the desired "minor" version of Kubernetes, like `1.8` or `1.7`.
+
+| Name | Workload | Timeframe |
+|------|----------|-----------|
+| Pre-production | Development or testing environments running real workloads, with the goal of catching capability bugs early | Available upon release |
+| Production | Clusters serving any amount of production traffic and/or have high reliability and uptime requirements | Promoted 2-4 weeks after initial release |
+
+Configure your desired update channel in the Cluster Settings screen in the Console.
+
+## Upgrading between minor versions of Kubernetes
+
+Before attempting an upgrade to a new "minor" version, ensure you are running the latest Tectonic version of the current minor version. For example, you need to be running the latest `1.7.9-tectonic.3` before upgrading to `1.8.4-tectonic.1`.
```

Comment:
For example, you must be running `1.7.9-tectonic.3` before upgrading to the next available version, `1.8.4-tectonic.1'.